app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ActiveRecord::Base
+  has_one :posts
```

Comment:
- [ ] **Required:** User, in this case, refers to the post's author. When a post is created, an author i.e user can have more than one post in their account. I hope this clarifies it. Kindly update to `has_many :posts`.